Added the directory with my README.md file. Also made a very modest c…

vbsmbb opened the pull request:
…orrection to the main README.md file of the project.

# Description

- [x] I have added a folder(vbsmbb)
- [x] I have added a Readme.md file inside the folder
- [x] I understand the GitHub Forking Concept fully

atapas reviewed (changes requested):
@vbsmbb Sir I see also updated the main readme file that creating conflicts. Can you please take care?

vbsmbb commented:
Mr. Adhikary,

I had trouble revoking the changes so I deleted the readme file from my
fork, and then deleted my repository to start over. When I forked the
repository, again, it was missing the readme file. I am very sorry. I'll
leave things alone.

Sorry, again,
MikeS


On Tue, Nov 29, 2022, 7:14 AM Tapas Adhikary ***@***.***>
wrote:

> ***@***.**** requested changes on this pull request.
>
> @vbsmbb <https://github.com/vbsmbb> Sir I see also updated the main
> readme file that creating conflicts. Can you please take care?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/atapas/fork-me/pull/429#pullrequestreview-1197436606>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AJYRKZI4DIREQPQ6IXQP6A3WKXXQRANCNFSM6AAAAAASJHDOFA>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>
